### PR TITLE
slove jenkins job partial failed no display problem

### PR DESCRIPTION
**Pull request by shiywang**
~I am still testing my result will paste log soon~

https://localhost:8888/job/hack/job/shiywang-aos-cd-jobs/job/build%252Fupdate-base-images/3/console

### CODE REVIEW

File: jobs/build/update-base-images/build.sh [hunk 1]
```diff
@@ -1,5 +1,8 @@
 #!/usr/bin/env bash
 
+set -e
```
Comment (shiywang) [inline jobs/build/update-base-images/build.sh]: I belive comman.shell() already set this up, but from the jenkins log I didnt see any setup... so just add it here to double-check

**Comment by shiywang** [comment]
@sosiouxme  PTAL

File: jobs/build/update-base-images/build.sh [hunk 2]
```diff
@@ -1,5 +1,8 @@
...
+set -o pipefail
```
Comment (tbielawa) [inline jobs/build/update-base-images/build.sh]: We should standardize on

`set -euxo pipefail`

https://vaneyckt.io/posts/safer_bash_scripts_with_set_euxo_pipefail/

**Comment by tbielawa** [comment]
lgtm after that shell script gets updated

**Comment by shiywang** [comment]
Thanks for the comments, I am going to merge this and try it out

**Pull request merged**